test/onnx/exporter/test_compat.py

```diff
@@ -86,6 +107,577 @@ def test__unflatten_dynamic_shapes_with_inputs_tree_succeeds_on_tuple_of_mixed_s
         )
         self.assertEqual(unflatten_dynamic_shapes, expected_dynamic_shapes)
 
+    @common_utils.parametrize(
+        "model, args, kwargs,input_names, output_names, dynamic_axes, expected_dynamic_shapes",
+        [
+            # llama-3.2-1B-Instruct
+            (
+                LlamaModelTest(),
+                (),
+                {
+                    "input_ids": torch.randn(2, 16),
+                    "attention_mask": torch.randn(2, 32),
+                    "position_ids": torch.randn(2, 16),
+                    "past_key_values": [
```

Comment:
Since 16 entries and 3 entries are effectively the same, we can shorten this test case a little

Reply:
Done